File: examples/c-fixtures/exits.c
/* Runs to completion and exits cleanly. The `state: exited` case. */
#include <stdio.h>

int main(void) {
    int x = 5;
    printf("about to finish x=%d\n", x); /* line 6 — breakpoint line */
    fflush(stdout);
    return 0;
}

File: examples/go-fixtures/exits.go
// Runs to completion and exits cleanly. The `state: exited` case.
//
// Line 11 is the print, which is the breakpoint line `wait_delve.rs` uses.

package main

import "fmt"

func main() {
	x := 5
	fmt.Printf("about to finish x=%d\n", x)
}
